docs/articles/phase-features-with-feature-flags.md

```diff
@@ -133,6 +133,9 @@ Aside from the licensing and maintenance cost of a feature flag service, you're
 * You'll need to identify and remove stale feature flags 
 * Understand and test the implications of flipping a feature flag
 
+> [!TIP]
+> To minimize the costs associated with the use of feature flags, keep feature flags short lived and prevent multiple feature flags from interfering with eachother by affecting the same functionality.
```

Comment:
@henrybeen can you add a space between each and other? "... with eachother by..."  should be "... with each other by …"

Reply:
@wpschaub  DONE